packages/components/organisms/f-checkout/src/checkout-issues.js

```diff
@@ -44,6 +44,11 @@ const checkoutIssues = {
         shouldRedirectToMenu: false
     },
 
+    DATE_OF_BIRTH_REQUIRED: {
```

Comment:
Get and update checkout have`AGE_VERIFICATION_REQUIRED` and `DATE_OF_BIRTH_REQUIRED` as issues, do we need to check for both?

Reply:
I was told it'll be `DATE_OF_BIRTH_REQUIRED`  but we will also need to handle `AGE_VERIFICATION_FAILED` when the user enters an invalid DOB.

Reply:
Can use the `DOB_REQUIRED_ISSUE` constant here?

```suggestion
    [DOB_REQUIRED_ISSUE]: {
```